src/classes/CW_BriefApprovalProcessTest.cls

```diff
@@ -0,0 +1,297 @@
+	    				   What_is_being_communicated__c,
+	    				   Target_Audience_Insight__c,
+	    				   Requestor_Department__c,
+	    				   Cost_Approval_Status__c,
+	    				   Urgent_Campaign__c
+	    			FROM CW_Campaign__c
+	    			WHERE Id =: camp.Id];
+	        
+	        CW_BriefApprovalProcess briefAP = new CW_BriefApprovalProcess();
+	        briefAP.submitBriefForApproval(camp);
+
+	        system.assertEquals(briefAP.errorMessage , ConstantUtil.SUBMIT_FOR_APPROVAL_BRIEF_NO_ROADMAP_APPROVED + roadmap.Approval_Status__c);
+        	
+     	 }
+    }
+    
+   	static testMethod void submitBriefForApprovalSuccesfully() 
+    {
+    	initUsersAndCustomSettings();
+    	
+    	CW_Brand__c brand = TestDataUtil.createBrand();
+        insert brand;
+        
+        CW_Label__c label = TestDataUtil.createLabel(brand.Id);
+        insert label;
+        
+        CW_Country__c country = TestDataUtil.createCountry();
+        insert country;
+        
+        CW_Product__c product = TestDataUtil.createProduct();
+        insert product;
+        
+        CW_Language__c language = TestDataUtil.createLanguage();
+        insert language;
+        
+        CW_Site_Catalogue__c siteCatalogue = TestDataUtil.createSiteCatalogue(label.Id, country.Id, product.Id);
+        insert siteCatalogue;
+    	
+     	CW_Campaign__c newCamp = TestDataUtil.createCampaign();
+     	insert newCamp;
+     	 	
+	    CW_Campaign_Sites__c campSites = TestDataUtil.createCampaignSite(newCamp.Id, siteCatalogue.Id, language.Id);
+       	insert campSites;
+    	
+    	CW_Roadmap__c roadmap = TestDataUtil.createRoadmap();
+    	roadmap.Approval_Status__c = 'Not Submitted';
+        insert roadmap;
+        
+    	updateCampRequiredFields(newCamp);	
+    	
+     	System.runAs(nonApproverUser)
+     	{
+     		newCamp.Name = 'New Campaign';
+     		newCamp.OwnerId = nonApproverUser.Id;
+     		newCamp.Stage__c = 'Roadmap Approved';
+     	 	newCamp.Cost_Approval_Status__c = 'Approved';
+	    	newCamp.Roadmap__c = roadmap.Id;
+	    	update newCamp;
+	    	
+	    	roadmap.Approval_Status__c = 'Approved';
+	    	update roadmap;
+	    	
+	        newCamp = [SELECT Name, Campaign_Start_Date__c,
+	        			   Campaign_End_Date__c, 
+	    				   Suggested_MarCom_Delivery_Date__c,
+	    				   OwnerId, 
+	    				   Main_Campaign_Business_Objective__c,
+	    				   Campaign_Focus__c,Roadmap__c,
+	    				   What_is_being_communicated__c,
+	    				   Target_Audience_Insight__c,
+	    				   Requestor_Department__c,
+	    				   Cost_Approval_Status__c,
+	    				   Urgent_Campaign__c
+	    			FROM CW_Campaign__c
+	    			WHERE Id =: newCamp.Id];
+	        
+	        CW_BriefApprovalProcess briefAP = new CW_BriefApprovalProcess();
+	        Boolean isSubmitted = briefAP.submitBriefForApproval(newCamp);
+
+	        system.assertEquals(isSubmitted , true);
+        	
+     	 }
+    }
+    
+    static testMethod void submitUrgentBriefForApprovalSuccesfully() 
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility